target/linux/ath79/dts/qca9533_tplink_tl-wr802n-v2.dts

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "qca9533_tplink_tl-wr802n.dtsi"
+
+/ {
+	model = "TP-Link TL-WR802N v2";
+	compatible = "tplink,tl-wr802n-v2", "qca,qca9533";
+
+	keys {
+		reset {
+			gpios = <&gpio 12 GPIO_ACTIVE_LOW>;
```

Comment:
Did you evaluate this yourself? In ar71xx, the v1 variant from this PR is used for both.

Despite, I'd prefer to have the full keys node in the DTS if keys actually differ, as this is much easier to read for only one button.

Reply:
Yes, I found this out because reset button with assignment taken from ar71xx wasn't working on V2. Once I had the system booted I could export all unassigned GPIOS and find this out by looking at /sys/kernel/debug/gpio.

Reply:
I'll also take a look on GPL sources for V1 and compare it against V2 - this should be present somewhere there. I can do at least that not having an actual V1 to test on.

Reply:
At least for V2 it's resolved, wonder what happens on mailing list with V1.